### DISCUSSION TITLE
url escape?

### DISCUSSION THREAD

**Discussion by avoidik** [mikefarah/yq#1357] [Q&A]
hello,

is there any alternative in yq to this thing in jq? the goal is to escape special symbols from a password

```bash
MYPASS=`echo -n "$ORIGPASS" | jq -sRr @uri`
```

**Comment by mikefarah** [comment]
Only way I can think of is to use a regular expression replacement: https://mikefarah.gitbook.io/yq/operators/string-operators#substitute-replace-string-with-regex
